src/components/sheets/ViewReactionsSheet.tsx

```diff
@@ -0,0 +1,121 @@
+import { useRef, useState, useContext, useEffect } from 'react';
+import { View, Text, Image, StyleSheet, TouchableOpacity } from 'react-native';
+import { ScrollView } from 'react-native-gesture-handler';
+import { observer } from 'mobx-react-lite';
+
+import type BottomSheetCore from '@gorhom/bottom-sheet';
+import { BottomSheet } from '@clerotri/components/common/BottomSheet';
+import { BottomSheetScrollView } from '@gorhom/bottom-sheet';
+import { app, setFunction } from '@clerotri/Generic';
+import { Message, User } from 'revolt.js';
+import { client } from '@clerotri/lib/client';
+import { commonValues, ThemeContext, Theme } from '@clerotri/lib/themes';
+import { MiniProfile } from '../common/profile';
+import { ReactionBox } from '../common/messaging/MessageReactions';
+
+type ReactionPile = {
+  emoji: string;
+  reactors: string[];
+};
+
+export const ViewReactionsSheet = observer(() => {
+  const {currentTheme} = useContext(ThemeContext);
+  const localStyles = generateLocalStyles(currentTheme);
+
+  const [message, setMessage] = useState(null as Message | null);
+  const [reaction, setReaction] = useState(null as string | null);
+
+  const sheetRef = useRef<BottomSheetCore>(null);
+
+  setFunction('openViewReactions', async (m: Message | null, emoji: string) => {
```

Comment:
unused async - also the types here don't quite match the Generic.tsx placeholder
```suggestion
  setFunction('openViewReactions', (m: Message | null, emoji: string | null) => {
```